src/Arcus.Observability.Tests.Integration/Serilog/Sinks/ApplicationInsights/AzureSearchDependencyTests.cs

```diff
@@ -30,7 +30,7 @@ public async Task LogAzureSearchDependency_SinksToApplicationInsights_ResultsInA
                 ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
 
                 bool isSuccessful = BogusGenerator.PickRandom(true, false);
-                DateTimeOffset startTime = BogusGenerator.Date.RecentOffset(days: 0);
```

Comment:
Is there a reason why we are changing this?

Reply:
For some reason, a lot of integration tests were failing.  It seemed like they were timing out because the dependencies that were written to AppInsights were not retrieved in time.   Sometimes I could reproduce it on my machine, sometimes not.
Then I decided to see what happens if I only retrieve the dependency-logs from AppInsights that occured in the last x minutes, by specifying a timespan filter.  Then, it looks like everything succeeds again.

Offcourse, to do this, I can't rely on random generated timestamps for the events.  I also suspect, but I have no proof for it, that writing a lot of things to appinsights with out-of-order timestamps might hurt AppInsights performance, as I would thing that AppInsights is timeseries based so some re-ordering might happen in appinisghts if you write out of order, but again, I only suspect that.
IMHO, it's also not a big deal to use the current datetime as the timestamp for the event that we're writing; it's actually as random as picking just another time for today's date.

Reply:
I think this is related to the fact that we have not updated the application insights sink yet which is not respecting the milliseconds.

Can you test the same after making the changes and also with the original recent offset please?

Reply:
I can't test it using the preview build in my project.  For some reason, it keeps complaining that Arcus.Observability.Correlation when I use the preview build in my project.

Even if I explicitly add all Arcus.Observability projects to my project, it keeps on complaining.
I also have a dependency on Arcus.WEbApi.Logging.Core; maybe they don't play well together ?

```
Could not load file or assembly 'Arcus.Observability.Correlation, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null'. 
The system cannot find the file specified.
```

Reply:
If I remove the Arcus.WebApi.Logging package from my project, I don't have the AssemblyLoadException anymore.

Reply:
> I think this is related to the fact that we have not updated the application insights sink yet which is not respecting the milliseconds.
> 
> Can you test the same after making the changes and also with the original recent offset please?

I've tested the current preview build in my project, and the timestamp of the dependency is correct up to the millisecond.
However, I see that the timestamps of all other traces in AppInsights have an even higher precision (it's in ticks).  Think we should also go to that precision.  I'll do some updates here.

Reply:
> I think this is related to the fact that we have not updated the application insights sink yet which is not respecting the milliseconds.

There are tests available which test if the date that is transformed by the appinsights converter is set correct.